projects/admin/src/app/acquisition/components/order/order-detail-view/order-detail-view.component.html

```diff
@@ -14,56 +14,87 @@
         <!-- ORDER DATE -->
         <dt class="col-4 label-title" translate>Order date</dt>
         <dd class="col-8">
-          <ng-container *ngIf="record.metadata.order_date else notYetOrdered">
-            {{ record.metadata.order_date | dateTranslate :'shortDate' }}
+          <ng-container *ngIf="order.order_date else notYetOrdered">
+            {{ order.order_date | dateTranslate :'shortDate' }}
           </ng-container>
           <ng-template #notYetOrdered>
             <i class="text-muted" translate>Not yet ordered</i>
           </ng-template>
         </dd>
         <!-- ORDER TYPE -->
         <dt class="col-4 label-title" translate>Order type</dt>
-        <dd class="col-8">{{ record.metadata.type | translate }}</dd>
+        <dd class="col-8">{{ order.type | translate }}</dd>
         <!-- ORDER STATUS -->
         <dt class="col-4 label-title" translate>Order status</dt>
-        <dd class="col-8">{{ record.metadata.status | translate }}</dd>
+        <dd class="col-8">{{ order.status | translate }}</dd>
         <!-- DESCRIPTION -->
-        <ng-container *ngIf="record.metadata.description">
+        <ng-container *ngIf="order.description">
           <dt class="col-4 label-title" translate>Description</dt>
-          <dd class="col-8">{{ record.metadata.description }}</dd>
+          <dd class="col-8">{{ order.description }}</dd>
+        </ng-container>
+        <!-- NOTES -->
+        <ng-container *ngIf="order.notes && order.notes.length > 0">
+          <dt class="col-4 label-title" translate>Notes</dt>
+          <dd class="col-8">
+            <a href (click)="scrollTo($event, 'notes'); notesCollapsed = false;" *ngFor="let note of order.notes">
+              <span class="badge {{ getBadgeColor(note.type) }} mr-2">{{ note.type | translate }}</span>
+            </a>
+          </dd>
         </ng-container>
-        <!-- CURRENCY -->
-        <dt class="col-4 label-title" translate>Currency</dt>
-        <dd class="col-8">{{ record.metadata.currency }}</dd>
       </dl>
     </div>
     <!-- ORDER TOTAL AMOUNT ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ -->
     <div class="col-4 pr-0">
-      <div class="border h-100 rounded bg-light text-center" *ngIf="totalAmount$ | async as totalAmount">
-        <span class="total-amount">{{ totalAmount | currency:record.metadata.currency:'symbol' }}</span>
+      <div class="border h-100 rounded bg-light text-center" *ngIf="order.total_amount">
+        <span class="total-amount">{{ order.total_amount | currency:order.currency:'symbol' }}</span>
       </div>
     </div>
   </section>
 
   <!-- ORDER LINES ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ -->
   <admin-order-lines
     class="row"
-    [order]="record"
+    [order]="order"
     [orderLines$]="orderLines$"
     (delete)="deleteOrderLine($event)"
   ></admin-order-lines>
+
+  <!-- ORDER NOTES ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ -->
+  <div class="row mt-3" *ngIf="order.notes && order.notes.length > 0">
+    <a id="notes"></a>
```

Comment:
What is the utility of this?

Reply:
It will create an anchor to link on this part of the screen.
If the order has a long list of order_lines, the notes are displayed far away at the bottom of the screen. But in the order summary (top of the screen) you have a badge to mention that's exists notes. If you click on this badge, the screen will be scrolled to this anchor tag.